tutors3/patches/openedx-common-settings

```diff
@@ -1,18 +1,17 @@
-AWS_S3_HOST = "{{ S3_HOST }}"
-AWS_S3_PORT = {{ S3_PORT }}
+DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+VIDEO_IMAGE_SETTINGS["STORAGE_KWARGS"]["location"] = VIDEO_IMAGE_SETTINGS["STORAGE_KWARGS"]["location"].lstrip("/")
+VIDEO_TRANSCRIPTS_SETTINGS["STORAGE_KWARGS"]["location"] = VIDEO_TRANSCRIPTS_SETTINGS["STORAGE_KWARGS"]["location"].lstrip("/")
+GRADES_DOWNLOAD["STORAGE_KWARGS"]["location"] = GRADES_DOWNLOAD["STORAGE_KWARGS"]["location"].lstrip("/")
+
+ORA2_FILEUPLOAD_BACKEND = "s3"
+FILE_UPLOAD_STORAGE_BUCKET_NAME = "{{ S3_STORAGE_BUCKET }}"
+
+AWS_S3_SIGNATURE_VERSION = "{{ S3_SIGNATURE_VERSION }}"
+AWS_S3_ENDPOINT_URL = "{{ "https" if ENABLE_HTTPS else "http" }}://{{ S3_HOST }}:{{ S3_PORT }}"
 AWS_S3_USE_SSL = {{ "True" if S3_USE_SSL else "False" }}
 AWS_S3_SECURE_URLS = {{ "True" if S3_USE_SSL else "False" }}
-AWS_S3_CALLING_FORMAT = "boto.s3.connection.OrdinaryCallingFormat"
-AWS_AUTO_CREATE_BUCKET = {{ "True" if S3_AUTO_CREATE_BUCKET else "False" }}
```

Comment:
I understand that `AWS_AUTO_CREATE_BUCKET` is slated for removal from django-storages, and it defaults to `False`. What I'm not sure about is whether this is true for django-storages **1.8** (which Open edX still depends on). Even if we drop the `S3_AUTO_CREATE_BUCKET` config option, does it hurt if we explicitly set `AWS_AUTO_CREATE_BUCKET = False` here?

Reply:
In django-storages 1.8, `AWS_AUTO_CREATE_BUCKET` is [set to `False` by default](https://github.com/jschneier/django-storages/blob/1.8/storages/backends/s3boto.py#L210). But we can be explicit here as the [documentation for that version](https://django-storages.readthedocs.io/en/1.8/backends/amazon-S3.html) just says that is it optional and fails to mention what the default value is.

Reply:
Yes, let's be explicit and set it to `False`. That's also [the precedent set by `tutor-minio`](https://github.com/overhangio/tutor-minio/blob/master/tutorminio/patches/openedx-common-settings#L12).